src/apphosting/config.ts

```diff
@@ -243,6 +244,9 @@ export async function exportConfig(
   // update apphosting.local.yaml
   await localAppHostingConfig.upsertFile(localAppHostingConfigPath);
   logger.info(`Wrote secrets as environment variables to ${APPHOSTING_LOCAL_YAML_FILE}.`);
+
+  updateOrCreateGitignore(backendRoot, [APPHOSTING_LOCAL_YAML_FILE]);
```

Comment:
should this be projectRoot?

Reply:
I guess that comes down to preference. I'm guessing having a single .gitignore at project root is more common even in monorepos, so we can go with that?

Reply:
discussed offline to create or update .gitignore at projectroot